pages/_app.tsx

```diff
@@ -72,16 +72,17 @@ export default function PlanetWeb({ Component, pageProps, err }: any) {
     loadConfig();
   }, []);
 
-  React.useEffect(() => {
-    if (localStorage.getItem('language') !== null) {
-      i18n.changeLanguage(localStorage.getItem('language'));
-    } else {
-      i18n.changeLanguage('en');
-    }
-  }, []);
+  // Norbert: language gets detected by i18next-browser-languagedetector
```

Comment:
We do not need this. If browser detection does not work, i18n also stores 'en' as fallback.